### PR TITLE
Contact validation form

### PR DESCRIPTION
**Pull request by peterboro**
In this pull request I implemented the following changes: 

- Collaborated with a coding partner.
- Setup and run linters.
- Implement a simple validation to my form: The content of the email field has to be in lower case.
- You should implement the following interactions: When the user submits the form, you check if the email is in lower case.
- If the validation is OK, the form is sent.
- If the validation is not OK, you show an error message to the user near the submit button informing them of the error, and the form is not sent.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -192,4 +192,26 @@ btns.forEach((btn, index) => {
   btn.addEventListener('click', () => {
     popUp(index);
   });
-});
\ No newline at end of file
+});
+
+// eslint-disable-next-line array-bracket-spacing, comma-dangle
+const [form, ] = document.getElementsByClassName('contactform');
+const error = document.querySelector('.error');
```
Comment (tufoinnkuo10) [inline main.js:199]: - [ optional] kindly make your error message color red so it can easily be noticed

File: main.js [hunk 2]
```diff
@@ -192,4 +192,26 @@ btns.forEach((btn, index) => {
...
+const validEmailRegExp = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+// eslint-disable-next-line no-unused-vars
+const message = document.getElementById('message');
+form.addEventListener('submit', (e) => {
+  // eslint-disable-next-line no-unused-vars
+  const { name, email } = form.elements;
+  const message = [];
```
Comment (tufoinnkuo10) [inline main.js:206]: - [ ] for clean code and best JS practices, kindly remove the disable ESLint comments and fix the problems as they might affect your code ahead. 😄

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi Team

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅Form submits fine
✅ Email check well implemented

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Sahar-AbdelSamad** [comment]
## Status Approved 🎉 🥳 

Hi @peterboro , :wave::wave:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
✔️ Your project looks awesome
✔️ Linters are passing

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 2 above)
Comment (peterboro) [reply main.js]: I appreciate the review. I've made the changes requested.

**Comment by peterboro** [comment]
Thank you for the review and approval.

**Pull request merged**